cmd/tmsUpload_test.go

```diff
@@ -141,6 +144,27 @@ func (cim *communicationInstanceMock) UploadFileToNode(nodeName, fileId, descrip
 	}
 }
 
+func (cim *communicationInstanceMock) ExportFileToNode(nodeName, fileId, description, namedUser string) (tms.NodeUploadResponseEntity, error) {
```

Comment:
This is export relevant, it should not reside in the tmsUpload_test. Maybe the mock can have its own file?